supporter-product-data/cloudformation/cfn.yaml

```diff
@@ -328,7 +418,6 @@ Resources:
       AlarmName: !Join
         - ' '
         - - 'URGENT 9-5 -'
-          - !Ref 'Stage'
           - 'Failed to extract supporter product data from Zuora'
```

Comment:
in that case no need the join, not to mention now we have alarms mobbing there's no such thing as urgent is there?

`AlarmName: 'Failed to extract supporter product data from Zuora'`